.github/actions/mount-s3/action.yml

```diff
@@ -7,12 +7,12 @@ runs:
       shell: bash
       run: |
         sudo apt install s3fs
-        sudo mkdir /s3bucket
-        sudo chown -R $(id -u):$(id -g) /s3bucket
+        sudo mkdir $MOUNTED_FOLDER
```

Comment:
Spravne by to mal byt `input`, vid napr.
https://github.com/keboola/keboola-as-code/blob/edc4618f7d88a64d3c069ee6a58861a3dd1d2af0/.github/actions/go-cache/action.yml#L3-L6

Pouzite takto:
https://github.com/keboola/keboola-as-code/blob/edc4618f7d88a64d3c069ee6a58861a3dd1d2af0/.github/workflows/push.yml#L123-L124

Reply:
Ahá, ok. 👍 https://github.com/keboola/keboola-as-code/pull/891/commits/3bf5ad63ff9c5d7deaaa401dc8c1d8fbf58ef8fa